e2e/tests/nightly/runtime.spec.ts

```diff
@@ -27,4 +43,46 @@ describe('Updating runtime status', () => {
     // Delete runtime
     await runtimePanel.deleteRuntime();
   });
+
+  test('Create with detachable disk, re-attach, delete', async () => {
+    await createWorkspace(page);
+
+    const runtimePanel = new RuntimePanel(page);
+    await runtimePanel.open();
+    await runtimePanel.getCustomizeButton().click();
+
+    // Select detachable disk, start.
+    await runtimePanel.pickDetachableDisk();
+
+    // TODO(calbach): Investigate why 110GB -> 120GB causes issues.
+    await runtimePanel.pickDetachableDiskGbs((await runtimePanel.getDetachableDiskGbs()) + 10);
+    await runtimePanel.createRuntime({ waitForComplete: false });
+
+    // Run notebook to write a file to disk.
+    let dataPage = new WorkspaceDataPage(page);
+    let notebookPage = await dataPage.createNotebook(makeRandomName('disk-before'));
+    await notebookPage.uploadFile(diskBeforeNotebookName, diskBeforeNotebookFilePath);
+    let codeOutput = await notebookPage.runCodeFile(1, diskBeforeNotebookName);
+    expect(codeOutput).toMatch(/success$/);
+
+    await notebookPage.save();
+    dataPage = await notebookPage.goDataPage();
+
+    // Select increase detachable disk, enable GPU to force a recreate.
+    await runtimePanel.open();
+    await runtimePanel.pickDetachableDiskGbs((await runtimePanel.getDetachableDiskGbs()) + 10);
+    await runtimePanel.pickEnableGpu();
+    await runtimePanel.applyChanges();
+
+    // Run notebook to verify file is still on disk; check new disk size.
+    notebookPage = await dataPage.createNotebook(makeRandomName('disk-after'));
+    await notebookPage.uploadFile(diskAfterNotebookName, diskAfterNotebookFilePath);
+    codeOutput = await notebookPage.runCodeFile(1, diskAfterNotebookName);
+    expect(codeOutput).toMatch(/success$/);
+
+    // Delete runtime, then delete disk
+    await runtimePanel.open();
+    await runtimePanel.deleteRuntime();
+    await runtimePanel.deleteUnattachedPd();
```

Comment:
Should test also delete `disk-after` and `disk-before` notebooks? Maybe we want to keep them around in case checking on them manually

Reply:
I've been going back to them, I don't see a strong need to delete these.